test/test_torch.py

```diff
@@ -2909,7 +2909,7 @@ def test_gradient_extreme_cases(self, device, dtype):
         large_size = 100000
         t = make_tensor((large_size,), dtype=dtype, device=device)
         t_np = t.cpu().numpy()
-        coordinates_np = list(np.random.randn(large_size))
+        coordinates_np = np.random.randn(large_size)
```

Comment:
what's the issue with casting it to a list?

Reply:
Original numpy : `coordinates_np` becomes a list of array scalars; the next line, `tensor(coordinates_np)`,  converts it into a tensor.
Under dynamo: `coordinates_np` becomes a list of 0D `torch.ndarray`s; the next line, `tensor(coordinates_np)` fails with "len() of an unsized object".

Reply:
rrright. Thank you!